SparseGrids/tasgridTestFunctions.cpp

```diff
@@ -80,6 +80,14 @@ TwoOneCosCos::TwoOneCosCos(){} TwoOneCosCos::~TwoOneCosCos(){} int TwoOneCosCos:
 const char* TwoOneCosCos::getDescription() const{ return "f(x,y) = cos(pi/2 * x) cos(pi/2 * y)"; }
 void TwoOneCosCos::eval(const double x[], double y[]) const{ y[0] = cos(0.5 * M_PI * x[0]) * cos(0.5 * M_PI * x[1]); } void TwoOneCosCos::getIntegral(double y[]) const{ y[0] = 16.0 / (M_PI * M_PI); }
 
+TwoOneExpSinCos::TwoOneExpSinCos() {} TwoOneExpSinCos::~TwoOneExpSinCos() {} int TwoOneExpSinCos::getNumInputs() const{ return 2; } int TwoOneExpSinCos::getNumOutputs() const{ return 1; }
+const char* TwoOneExpSinCos::getDescription() const{ return "f(x,y) = exp(sin(2*pi*x) + cos(2*pi*y)) on [0,1]^2 "; }
+void TwoOneExpSinCos::eval(const double x[], double y[]) const{ y[0] = exp(sin(2*M_PI*x[0])+cos(2*M_PI*x[1])); } void TwoOneExpSinCos::getIntegral(double y[]) const{ y[0] = 1.6029228068079633; }
+
+TwoTwoSinCos::TwoTwoSinCos() {} TwoTwoSinCos::~TwoTwoSinCos() {} int TwoTwoSinCos::getNumInputs() const{ return 2; } int TwoTwoSinCos::getNumOutputs() const{ return 2; }
+const char* TwoTwoSinCos::getDescription() const{ return "f(x,y) = [sin(2*pi*x)cos(4*pi*y), cos(2*pi*x)]"; }
+void TwoTwoSinCos::eval(const double x[], double y[]) const{ y[0] = sin(2*M_PI*x[0])*cos(4*M_PI*x[1]); y[1]=cos(2*M_PI*x[0]); } void TwoTwoSinCos::getIntegral(double y[]) const{ y[0] = 0.0; y[1]=0.0; }
```

Comment:
* `cos(2 * M_PI * x)` is too simple, try something that doesn't belong to the basis